data/human/free worlds 1 start.txt

```diff
@@ -1015,7 +1015,7 @@ mission "FW Senate 1"
 		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
 	
 	on offer
-		log "The Free Worlds have finally elected a Senate, which will be convening for the first time soon on Bourne."
+		log "The Free Worlds has finally elected a Senate, which will be convening for the first time soon on Bourne."
```

Comment:
Again, literally referring to the individual worlds.

Reply:
```suggestion
		log "The Free Worlds have finally elected a Senate, which will be convening for the first time soon on Bourne."
```